info.yaml

```diff
@@ -8,10 +8,10 @@ project_category: 'metadata'
 project_lead: &odpi_releng_ptl
     name: 'Mandy Chessell'
     email: 'mandy_chessell@uk.ibm.com'
-    id: 'mandy-chessell'
+    github_id: 'mandy-chessell'
     company: 'IBM'
     timezone: 'Europe/London'
-primary_contact: *odpi_releng_ptl
+primary_contact: *odpi_egeria_ptl
 issue_tracking:
     type: 'github'
     url: https://github.com/odpi/egeria/issues'
```

Comment:
The url for issue_tracking seems to have a dangling single quote?  Noticed the same under the list of repos (maybe worth scanning for any other spots as well?)